elastic/kibana
# Only render context sections when asset criticality enabled

Removes all references to the "context" risk input category when the asset criticality experimental feature flag is disabled.

<img width="1713" alt="Screenshot 2024-02-12 at 12 38 40 AM" src="https://github.com/elastic/kibana/assets/147995946/4191119c-99c7-48bb-add5-d8c6678768c7">

## Testing Locally

To test this locally:
- Enable the `newUserDetailsFlyout` and/or `newHostDetailsFlyout` experimental flags, but do not enable the `entityAnalyticsAssetCriticalityEnabled` experimental flag. 
- Ensure that risk scores are visible (enable the risk engine, etc.)
- Navigate to the new entity flyout, and open the risk inputs expansion
